Resolver update

jeffwilcox opened the pull request:
- Environment npm package support

jeffwilcox committed:
Brainstorming

jeffwilcox committed:
Adding VSCode launch file

jeffwilcox committed:
Removing function resolver (it was confusing)

jeffwilcox committed:
Updated dependencies

jeffwilcox committed:
Inc. version

jeffwilcox committed:
v0.2.0: support for environment packages
Updates dependencies and introduces the ability for environment variables to
come from environment-based defined or discovered npm packages. Designed for
private feed NPM support while keeping config-as-code as a practice.

jeffwilcox committed:
Removing unused files

msftclas commented:
@jeffwilcox,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

jeffwilcox committed:
Removing vscode launch file

jeffwilcox committed:
Multi-graph with merge implementation
Enables configuration graph objects to be dynamic, fixed, etc., located:
- via a directory in the app and a configuration or env variable
- config packages defined in package.json
- config packages defined in environment variables dynamically/outside of package.json

jeffwilcox committed:
Minor comment change

jeffwilcox committed:
Returning pkg require

jeffwilcox committed:
Implementing package index path skip

jeffwilcox committed:
Linting/new dep
